tests/deploy/api/test_batch_endpoints.py

```diff
@@ -4,7 +4,7 @@
 from ragna.deploy import Config
 from ragna.deploy._api import app
 
-from .utils import authenticate
+from ..utils import authenticate
```

Comment:
Let's not have relative imports that go beyond the current namespace

```suggestion
from tests.utils import authenticate
```